web/satellite/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Disallow:
+Disallow: /cgi-bin/
```

Comment:
Instead of serving this from a file, I think we can hard-code this.

Reply:
Why do we need to add production level settings to our public development codebase?

Reply:
Yeah, I was thinking of reducing possibilities of getting the robots file wrong. But I guess it doesn't matter that much.

This robots won't matter for dev environment.

But, yeah, I think the implementation might end up simpler if we don't have it.

Reply:
this is the very base implementation of robots.txt file. The content of robots.txt could be taken from config in future

Reply:
we already have a few reasons to modify config of satellite server and we can include this one also. But having "/robots.txt" path is a standard metric for web application

Reply:
I was thinking that this endpoint can be opened by some `proxy`, like we expose a server through `https proxy` to outer world. As `SEO` settings is individual for every `satellite owner` (every satellite is indexed differently) it should be configured as deployments step but not here.